### ISSUE TITLE
Feature Request: New category/tag on add modal

### ISSUE THREAD

**Issue by zackchadwick** [ndom91/sveltekasten#4]
### Clear and concise description of the problem

I'd like to be able to create a new tag or category within the prompt for adding a new bookmark.

Not sure if this is on your radar already or not.

### Suggested solution

Unsure on suggested solution at the moment. 

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ndom91** [comment]
Makes sense, I've been meaning to add some more tag / category related UI in the bookmark / rss feed list.

Maybe an additional bar above the entries where one can filter, and then also create new ones, without having to navigate to the settings.
